src/network/unsigned_tx/eip712/mod.rs

```diff
@@ -330,7 +346,7 @@ impl SignableTransaction<Signature> for TxEip712 {
         // Drop any v chain id value to ensure the signature format is correct at the time of
         // combination for an EIP-1559 transaction. V should indicate the y-parity of the
         // signature.
-        let signature = signature.with_parity_bool();
+        let signature = signature.with_parity(true);
```

Comment:
Is this correct? Seems weird that we set parity to always be `true`, it might work for the specific example we have but generally it should be varied, no?

Would removing this line (along with the comment) not work here?